Libs/MRML/Core/vtkMRMLSliceNode.h

```diff
@@ -78,11 +78,27 @@ class VTK_MRML_EXPORT vtkMRMLSliceNode : public vtkMRMLAbstractViewNode
   vtkGetMacro(SliceVisible, int);
   vtkSetMacro(SliceVisible, int);
 
+  ///
+  /// The visibility of the slice edge in the 3DViewer.
+  vtkGetMacro(ThreeDSliceEdgeVisibility, int);
+  vtkSetMacro(ThreeDSliceEdgeVisibility, int);
+
+  ///
+  /// The opacity of the slice edge in the 3DViewer.
+  vtkGetMacro(ThreeDSliceEdgeOpacity, double);
```

Comment:
Opacity and and radius should not be necessary to tune for each view. It should "just work", the slice view should just look nice. The same scaling factor that is used for relative scaling of markups could be used here (`this->GetScreenScaleFactor() * this->ViewScaleFactorMmPerPixel`). Opacity could be set to 1.0.